DigitalLearningSolutions.Web/Helpers/SelectListHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.AspNetCore.Mvc.Rendering;
+
+    public static class SelectListHelper
+    {
+        public static IEnumerable<SelectListItem> GetOptionsWithSelectedText(List<(int id, string name)> options, string? selectedName)
+        {
+            var selectedOption = options.Where(o => o.name == selectedName).Select(o => o.id).SingleOrDefault();
```

Comment:
Given that there's a few things happening here, could we format it such that every data transformation is on a new line to improve readability?
```
options.Where(o => o.name == selectedName)
.Select(o => o.id)
.SingleOrDefault();
```